specification/flag-evaluation/hooks.md

```diff
@@ -74,21 +77,22 @@ EvaluationContext|void before(HookContext, HookHints)
 
 ##### Requirement 4.3.4
 
-> When `before` hooks have finished executing, any resulting `EvaluationContext`
-> **MUST** be merged with the invocation `EvaluationContext` with the invocation
-> `EvaluationContext` taking precedence in the case of any conflicts.
+> When `before` hooks have finished executing, any resulting
+> `EvaluationContext` > **MUST** be merged with the invocation
```

Comment:
This has an intermixed `>` character. May want to ensure the rest of these don't have that too.